### ISSUE TITLE
failed to read protocol/Cargo.toml

### ISSUE THREAD

**Issue by jedahan** [bytesnake/hex#21]
```zsh
git clone https://github.com/bytesnake/hex.git
cd hex/server
cargo build
# error: failed to read `~/code/hex/server/protocol/Cargo.toml`
```
Not sure why it is looking for `protocol/Cargo.toml`

**Comment by jedahan** [comment]
Removing https://github.com/bytesnake/hex/blob/master/Cargo.toml#L7 at least let me start building

**Comment by bytesnake** [comment]
Hey, I'm introducing a protocol crate right now and accidentally uploaded the new Cargo.toml. Sorry :D 
Fixed in 46cab3dfce3a609f129e82843a05586886bee0ff.

**Comment by jedahan** [comment]
Thanks for the quick response. Excited to make my own tokens :)
